smallrye-llm-langchain4j-liberty-bundle/BOM/pom.xml

```diff
@@ -0,0 +1,26 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
I don't think you need to check in the user feature as you do not use it.

Reply:
Done!